src/matUtils/README.md

```diff
@@ -9,59 +9,71 @@ matUtils is a toolkit for working with mutation annotated tree protobuf files, t
 
 **-h**: Print help messages
 
-## Annotate
+## Extract
 
-The annotate command takes a MAT protobuf file and a two-column tab-separated text file indicating sample names and lineage assignments. The software will automatically identify the best clade root for that lineage and save the assignment to each sample indicated, returning a new protobuf with these values stored.
+This command selects subtrees from the MAT based on clade identification, sample choice files, or placement quality metrics, and optionally converts to other file formats, including vcf and newick.
 
-### Specific Options
+**-s**: Select samples to extract by explicitly naming them, one per line in a text file.
 
-**-o**: Output mutation-annotated tree file (REQUIRED)
+**-c**: Select samples to extract by membership in the indicated clade.
 
-**-l**: Provide a path to a file assigning lineages to samples to locate and annotate clade root nodes (REQUIRED)
+**-m**: Select samples to extract by whether they contain the indicated mutation.
 
-**-f**: Set the minimum allele frequency in samples to find the best clade root node (default = 0.9)
+**-e**: Select samples to extract by whether they have less than the indicated number of equally parsimonious placements. Note: adds significantly to runtime with large sample selections.
 
-## Convert 
+**-a**: Select samples to extract by whether their parsimony score (terminal branch length) is less than the indicated value.
 
-The convert subcommand yields non-protobuf format files representing the tree. This can be a newick format file and/or a VCF representing each sample's variations from the reference. 
+**-r**: Automatically select two samples per unique clade identifier to extract.
 
-### Specific Options
+**-p**: Prune the samples indicated by other arguments instead of extracting them to create the subtree.
 
-**-v**: Output VCF file 
+**-d**: Set the directory to save output files. Directory must already exist. Default is current directory
 
-**-t**: Output Newick tree file
+**--sample-paths**: Write the path of mutations defining each selected sample to a file.
```

Comment:
You're using a mix of long form and short form command line options. Can we have both in all cases?

Reply:
I'd defined some that I don't anticipate users using very often with just longform because I was kind of running out of reasonable letters. I can give them shortforms as well, just might be a lot of -x and -b... maybe if I use some uppercases

Reply:
Yeah, no problem. There are just 52 total letters (upper- and lower-case). Choose whatever seems reasonable but let's have descriptive and shorthand notations here.

Reply:
its a good call/feedback, I didn't really think as much about it as I should have, better now